package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "image-blob-reduce",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Please don't touch version number, that's part of release prosess, not of this commit.

Reply:
removed